index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;200;300;400;500;700&family=Poppins:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+      type="text/css"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header>
+      <nav class="header">
+        <a href="#" class="logo">Abimbola</a>
+        <a><i class="fa fa-bars cursor"></i></a>
+      </nav>
+    </header>
+    <section class="main-section img">
+        <div>
+          <h1 class="greetings">Hey There.</h1>
+          <h2 class="greetings">I'm Abimbola</h2>
+          <h3 class="title">I'am a Software Developer</h3>
+          <p class="paragraph">
+            I can help you build a product, feature or website. Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, dont hestiate to contact me.
+          </p>
+        </div>
+        <ul class="flex">
+            <li>
+                <a href="https://github.com/bimbolabuari/"
+                target="_blank"
+                rel="noopener"><i class="fa fa-github icon"></i>
+                </a>
+            </li>
```

Comment:
Hello @Dipeshtwis, Please I would like you to share with me the advantage of `li` tag over `a` tag to group the icon so as to take note next time 👍

Reply:
Adding `ul` and `li` would make the code more structured and readable. As we are making a list item here. Hope this will make sense to you?